debian/changelog

```diff
@@ -1,3 +1,126 @@
+qbs (1.13.1-2) unstable; urgency=medium
```

Comment:
In order to release to the xenial repository, we'll need a changelog entry for that release.

Reply:
Fixed!